app/scripts/templates/settings/account_recovery/confirm_password.mustache

```diff
@@ -0,0 +1,27 @@
+<div id="account-recovery-confirm-password">
+    <section class="modal-panel">
+        <div class="error"></div>
+
+        <h2>{{#t}}Re-enter password{{/t}}</h2>
+
+        <form novalidate>
+            <p class="description">
+                {{#t}}Enter your password to view your recovery key.{{/t}}
```

Comment:
Some thoughts about this screen.

1. The "Enter your password to view your recovery key" looks out of place up there and is almost totally detached from the input element.
2. The email looks exceptionally heavy.
3. The important text kinds gets lost. It might be worthwhile to experiment [with the Photon styles for warnings](https://design.firefox.com/photon/patterns/warnings.html) to see if the yellow color and warning icon work well here.

<img width="402" alt="screen shot 2018-07-31 at 15 45 59" src="https://user-images.githubusercontent.com/848085/43469586-69646a2e-94de-11e8-9cca-550615d24197.png">

Here's a quick mock changing the order, making the email an h2, removing the current "re-enter your password to...." text, and adding the yellow color (I'm not being ambitious enough to try the icon):

<img width="463" alt="screen shot 2018-07-31 at 16 33 16" src="https://user-images.githubusercontent.com/848085/43470055-8e90078a-94df-11e8-8d94-930bd9dd527c.png">

Reply:
@shane-tomlinson This differs a lot from originally proposed UX in Phase 1.5, I can try to get some additional info from mliang. In meantime, I can flag it in https://github.com/mozilla/fxa-content-server/issues/6394 for follow up.

Reply:
@vbudhram I don't have the "important text" part in my UX spec, do you know where that came from?

I agree that the visual hierarchy could be better (with the important message) but I think the photon style yellow background is not suitable here since the message here is not "warning" to me and it might unnecessarily alert the users.

If we do need that important message, I can provide an updated version of this screen.

Reply:
@youwenliang I can't seem to find the version of UX that contained this. Do you have a final UX doc with copy changes? Best I have is [this](https://mozilla.invisionapp.com/share/RMMYSS8C4Z3#/screens/307726824).

In the meantime, I can remove the important block.